presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -600,6 +601,18 @@ public static long random(@SqlType(StandardTypes.BIGINT) long value)
         return ThreadLocalRandom.current().nextLong(value);
     }
 
+    @Description("a pseudo-random number from a normal distribution with mean, std, and p")
```

Comment:
What's p? If I'm reading this correctly, this function produces a normally-distributed value given a uniformly-distributed value `p`. If so, rand_normal doesn't seem like the right name for it.

Also, given the above interpretation (and from looking at what the function does), it shouldn't be marked as non-deterministic.

Reply:
Oh, just read the PR description. That explains it. So, we need to come up with a name that accurately describes: "Compute the inverse of the Normal cdf with given mean and standard deviation for the cumulative probability: P(N < n)."

Reply:
Maybe something like "inverse_normal_cdf"